libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,130 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface HubspotFormField {
+    name: string;
+    value: string | any;
+    click: () => void;
+}
+
+/**
+ * @public
+ */
+export interface IConversionTouchPointDialogBaseProps {
+    selectedValue?: string;
+    userEmail: string;
+    clientId: string | number;
```

Comment:
It seems that `clientId` is used to set value for `gacid` (something relates to Google Analytics, doesn't it?)
If the customers who use this component don't use Google Analytics (or if GA is forbidden by a cookie policy), they probably don't have any value for `clientId`, do they?
→Should we mark this field as optional?

Reply:
I have updated this to make it more general. Thanks!

Reply:
Should we keep the `clientId` field and the logic to set value for `gacid`? I think that, if we need to do some telemetry to know how many times users have used the conversion dialog, it would be convenient for analysis purpose if we can correlate which gaID has submitted the forrm.

※ The same logic to set value to `gacid` before submitting a Hubspot form can be found in gdc-website as well: https://github.com/gooddata/gdc-website/blob/92a613b399094ecc4fd6f75b58e6e22f8982474e/src/gdc/_includes/js/forms/hs.generic.js#L46

Reply:
You can do this via the `values` prop. So if you want to populate values for the form, you can set the values like this:
```javascript
values={{ email: 'abc@gmail.com', gacid: clientId }}
```